dbt/sagerx/models/staging/rxclass/stg_rxclass__has_pe.sql

```diff
@@ -0,0 +1,13 @@
+-- stg_rxclass__has_pe.sql
+
+with has_pe as (
+    
+    select
+        *
+    from {{ source('rxclass', 'rxclass') }}
+    where rela = 'has_pe'
+)
+
+select distinct
```

Comment:
I think maybe we move the distinct up to the cte above. The last part of every dbt model is generally supposed to be just like select * from final_cte (doesn't have to be called final_cte, but it's supposed to be easy to use to switch to other ctes in the model and having a distinct here just feels slightly "non-dbt" and I'm a little OCD about that. We need to add to our style guide...